docs/demultiplexing.md

```diff
@@ -160,6 +160,52 @@ Example configuration file for pipeline with `gather_from_node` specified before
 }
 ```
 
+## Dynamic batch handling with demultiplexing
+
+Demutliplexing feature enables handling requests with dynamic batch size. For workloads that involve frequent batch size change this is recommended way to handle such requests.
```

Comment:
dynamic batch size without a model reloading.

It is recommended for workloads with arbitrary batch size in the sequential requests.